src/config.ts

```diff
@@ -250,6 +250,11 @@ log.info(
   IDP_METADATA_REFRESH_INTERVAL_SECONDS
 );
 
+// LolliPoP protocol configuration params
+export const DEFAULT_LOLLIPOP_ASSERTION_REF_DURATION = (3600 *
```

Comment:
why we never try to retrieve this conf from env?
If we want to protect this value and change it only via PR, maybe we can move it outside the config.ts

Reply:
IMHO is better doesn't have this parameters configurable from env to avoid inconsitency, for example values grater than the storage time limit for assertions. Other default values are defined in `config.ts` so for coherence I put in here the parameter.